buildenv/docker/mkdocker.sh

```diff
@@ -502,6 +548,31 @@ fi
   echo "    xvfb \\"
   echo "    zip \\"
   echo "    zlib1g-dev \\"
+if [ $criu != no ] ; then
+  echo "    asciidoc \\"
+  echo "    iproute2 \\"
+  echo "    libaio-dev \\"
+  echo "    libbsd-dev \\"
+  echo "    libcap-dev \\"
+  echo "    libcap2-bin \\"
+  echo "    libnet1-dev \\"
+if [ $version = 20.04 ] ; then
+  echo "    libnftables-dev \\"
+fi
+  echo "    libnl-3-dev \\"
+  echo "    libprotobuf-c-dev \\"
+  echo "    libprotobuf-dev \\"
+  echo "    nftables \\"
+  echo "    protobuf-c-compiler \\"
+  echo "    protobuf-compiler \\"
+  echo "    python-ipaddress \\"
+  echo "    python-protobuf \\"
+if [ $version != 16.04 ] ; then
+  echo "    python3-distutils \\"
+fi
+  echo "    python3-future \\"
+  echo "    xmlto \\"
+fi
```

Comment:
Except possibly for 20.04, just installing the `criu` package should be sufficient for Ubuntu.

Reply:
Is not the required version. This provides the same version on all platforms.

Reply:
These packages are required to compile and run libcriu.

Reply:
On my Ubuntu 18.04 system, I have the criu package installed, but none of
```
libbsd-dev
libcap-dev
libgnutls28-dev
libnet1-dev
libnl-3-dev
libprotobuf-c-dev
libprotobuf-dev
protobuf-c-compiler
protobuf-compiler
python-ipaddress
```
I conclude those packages are not required to run applications that use criu.

I would much prefer that the package manager be used as opposed to building from source.